c/anagrams_kata/anagrams_finder.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ANAGRAMS_FINDER_H
+#define ANAGRAMS_FINDER_H
+
+typedef struct anagrams_finder_s
+{
+    char *wordlist;
+    void (*find)(struct anagrams_finder_s *_this, char *word);
```

Comment:
Yes you a right, in my 'implementation' the file is not loaded, but to be honest, this was on purpose. Just 2 points on this:

1. It was never specified that the file has to be pre loaded. 
2. I usually find it a bad idea to load the whole file at once, as it woult require massive memory on huge files.

In my solution I would try to find the anagrams on demand, without preloading the whole file. Sounds a bit like a raw solution but it would work regardless of the file size.

I think it is important to also address such issues even in a kata or exactly for that very reason.

Sorry, I will not come to the Dojo today, but I may attend in the future Dojos, as I find it quite interesting to discuss low level issues with 'high level' programmers ;)